src/App.js

```diff
@@ -11,7 +11,9 @@ import LoginPage from './pages/LoginPage/LoginPage';
 import SignUpPage from './pages/SignUpPage/SignUp';
 import CartPage from './pages/CartPage/CartPage';
 import { useAuth } from "./context/AuthContext";
-import { ROUTE_PATH_CartPage, ROUTE_PATH_LandingPage, ROUTE_PATH_LoginPage, ROUTE_PATH_Mockman, ROUTE_PATH_ProductListPage, ROUTE_PATH_SignupPage, ROUTE_PATH_Unknown, ROUTE_PATH_WishListPage } from "./utils/Routes"
+import { ROUTE_PATH_CartPage, ROUTE_PATH_LandingPage, ROUTE_PATH_LoginPage,ROUTE_PATH_ProductContentPage, ROUTE_PATH_Mockman, ROUTE_PATH_ProductListPage, ROUTE_PATH_SignupPage, ROUTE_PATH_Unknown, ROUTE_PATH_WishListPage } from "./utils/Routes"
```

Comment:
Suggestion: here instead of importing all the route paths you could just create an object with all the route path name as key and paths as values, and import just the object.

For example: 
`const routePath = {
     cartPage: "/cart",
}`

Use it as:
` path ={routePath.cart}`

Reply:
i guess it just double work maintaining everything in a object